subprojects/docs/src/snippets/compositeBuilds/basic/groovy/composite/settings.gradle

```diff
@@ -1,4 +1,4 @@
-rootProject.name = 'adhoc'
+rootProject.name = 'my-composite'
 
 includeBuild '../my-app'
 includeBuild '../my-utils'
```

Comment:
We should look if we can reuse the same templates for the projects used by `javaApplication` or one of the java multi-project samples.

Reply:
It's similar but not entirely. There are transitive project/binary dependencies in the composite build snippet, that don't exist in the `javaApplication` template. I'm not saying we shouldn't do that, it's just a lot more effort than it seems at first.

Reply:
I stand by this. It's too much effort to reuse the template for little reward. I'd keep the sample as it is now.